### ISSUE TITLE
Edge Endpoint "|"

### ISSUE THREAD

**Issue by iXanthos** [almende/vis#3412]
Hello, I would like to ask the addition of a new endpoint, the "|".

I saw #1993 , so I will explain briefly where this is used.

In biology, the up-regulation or stimulation is pointed out in the graphs by the use of the standard arrow
"->".
The down-regulation or inhibition, is depicted by the use of this endpoint "-| ".
It is a standard to separate up-regulation from down-regulation.
I've seen the code and it seems pretty easy (for you) to install an extra endpoint, so I hope I gave you a decent reason to add it.

tl;dr: Please add an edgepoint that is used by biologists in their graphs.

Thanks in advance.

**Comment by wimrijnders** [comment]
Γεια σου φίλε! Σπάνια βλέπω Έλληνες εδώ.

OK, the endpoint requests keep on stacking up. Would you mind given a graphical example of how the 'down-regulation' endpoint should look like? I suppose the current arrow is OK for your purposes.

I'll make a point up bumping endpoints to a higher priority. Unfortunately, my time is limited and I am incapable of keeping up with all the issues here, so forgive me if it should take longer than you would wish.

> I've seen the code and it seems pretty easy (for you) to install an extra endpoint,

I actually disagree with this, it's not easy enough. I intend to refactor this code to make it dirt-easy, so that you could have made the PR yourself instead of requesting the issue for this.

**Comment by wimrijnders** [comment]
Related to #3118.

**Comment by iXanthos** [comment]
Γεια σου και σένα!

Yeah, here I have attached what I would like to use. It shows the connection of 2 nodes in a graph. As stated above, the normal arrow is already used in the bibliography to signal the up-regulation, so it cannot be used for the down one as well, since it has to be distinct.

I called it easy, but used the keyword "seems", since I have no idea how the library actually works in its core, so pardon me.

![updownreg](https://user-images.githubusercontent.com/22717082/30005786-d0ac2b04-90f2-11e7-8e20-ef45479687f5.png)

**Comment by wimrijnders** [comment]
OK, that is clear enough. Is the shape of the arrow also important? I mean the gap that is in there.

I'll see what I can do, time permitting. In any case it's on the list.

**Comment by iXanthos** [comment]
No no, the arrow is just your already implemented one, the default. I just added it in order to give you a general idea of what the connections in those kinds of graphs look like. What is missing is the down-regulation part. 

Thanks for the help.

**Comment by wimrijnders** [comment]
Εντάξει, κατάλαβα/Okay, I understand.

It's just about  the down-regulation. As of now it's on the list.

**Comment by wimrijnders** [comment]
How does this look to you?

![download 5](https://user-images.githubusercontent.com/769037/30364522-db783522-9864-11e7-985a-4590557fcb25.png)

I named the endpoint `bar`, is that acceptable?

Also, I'm aware of overlapping issues with non-circular nodes. I'll leave it out of the context of this issue.

**Comment by iXanthos** [comment]
yeah its nice, I think it's clear enough. The name is not an issue, at least for me, so I guess this is not a problem overall.

**Comment by wimrijnders** [comment]
OK great. So the PR has been submitted. Ολα ενταξη!/All is well!
